bpf/profiling/network/args.h

```diff
@@ -244,9 +244,11 @@ static __inline struct socket_buffer_reader_t* read_socket_data(struct sock_data
     if (size > MAX_PROTOCOL_SOCKET_READ_LENGTH) {
         size = MAX_PROTOCOL_SOCKET_READ_LENGTH;
     }
+    // use & to work around an issue in kernel verifier.
+    // See https://github.com/apache/skywalking/discussions/10273
```

Comment:
Please put a summary in the comments directly.
We don't recommend to put a link in codes. The issue could be edited or be updated later.

Reply:
@wu-sheng 
I replace this with the original issue, so people can get rid of the workaround once the original issue is resolved.